test/storage-luatest/storage_1_1_test.lua

```diff
@@ -531,3 +531,79 @@ test_group.test_noactivity_timeout_for_explicit_master = function(g)
         _G.bucket_gc_wait()
     end, {g.replica_1_a:replicaset_uuid(), bid})
 end
+
```

Comment:
Need a storage test which boots a new cluster with names. Otherwise this is uncovered now, whether vshard is able to set the names for the new cluster. I suggest to do it in an new file `storage-luatest/persistent_names_test.lua`. (The test `test_group.test_named_config_identification` can remain in storage_1_1_test.lua, because requires the cluster to be booted without names.)

Also in that new file you can test the switch from names to UUIDs.

Reply:
Added the test. It's good, it was tested, `vtest.cluster_bootstrap` was broken, when name identification is used.